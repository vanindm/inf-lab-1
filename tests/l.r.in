8 5
1467.16437 9478.22714 3833.49407 1248.52097 8316.69305 
2135.78504 9701.45751 5322.29924 9825.95981 8405.42333 
5108.20323 9971.42591 729.92894 3900.18087 5705.37858 
7848.63896 6521.70111 3693.67314 6399.26286 9828.39297 
5135.80257 2529.09277 2201.11934 9681.23244 8264.27147 
4160.47412 8750.45803 2804.61923 2323.51321 6189.70068 
4811.36402 5108.25056 6677.18361 4931.83804 3350.20947 
5528.88546 3690.0016 732.42614 7616.79231 8721.41337 
4
5 6 1 2 3 4 5
