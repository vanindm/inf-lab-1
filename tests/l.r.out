1467.164370 9478.227140 3833.494070 1248.520970 8316.693050 
2135.785040 9701.457510 5322.299240 9825.959810 8405.423330 
5108.203230 9971.425910 729.928940 3900.180870 5705.378580 
7848.638960 6521.701110 3693.673140 6399.262860 9828.392970 
105463.121550 196278.185990 100204.383180 156360.156080 201219.447990 
4160.474120 8750.458030 2804.619230 2323.513210 6189.700680 
4811.364020 5108.250560 6677.183610 4931.838040 3350.209470 
5528.885460 3690.001600 732.426140 7616.792310 8721.413370 
